src/infra/src/cache/file_data/mod.rs

```diff
@@ -220,9 +220,9 @@ mod tests {
         cache.insert(key1.to_string(), 1);
         cache.insert(key2.to_string(), 2);
         cache.contains_key(key1);
-        cache.remove();
-        assert!(cache.contains_key(key1));
-        assert!(!cache.contains_key(key2));
+        cache.remove(); // contains_key() does not mark the key1 as used -> removed
```

Comment:
@haohuaijin please confirm if this is desired/acceptable behavior. The original test case wasn't passing and I figured that was the issue.

Reply:
You are right, at the beginning the LRU contains_key effect the result, but we fixed it😁
https://github.com/kyren/hashlink/issues/27